docs/index.md

```diff
@@ -160,20 +160,7 @@ To use the Red Hat Cloud Services provider inside your Terraform configuration y
                 "Sid": "VisualEditor1",
                 "Effect": "Allow",
                 "Action": [
-                   "s3:PutObject",
-                   "s3:GetObject",
-                   "s3:AbortMultipartUpload",
-                   "iam:ListRoles",
```

Comment:
all s3 actions are removed because added s3:*
Is it intended also removing iam and sts actions?

Reply:
updated the policy and forgot to add them to the index. Fixed